verifier/src/main/kotlin/tech/figure/classification/asset/verifier/client/VerifierClient.kt

```diff
@@ -96,10 +96,12 @@ class VerifierClient(private val config: VerifierClientConfig) {
             onBlock = { blockHeight ->
                 // Record each block intercepted
                 NewBlockReceived(blockHeight).send()
-                // Track new block height
-                latestBlock = trackBlockHeight(latestBlock, blockHeight)
             },
             onEvent = { event -> handleEvent(event) },
+            onEventsProcessed = { blockHeight ->
```

Comment:
I added this handler so a function could be called after a block's worth of events was processed, to avoid saving a height but then failing to process the events at that height (and subsequently restarting at the next height and never coming back)